src/offline/ledgerhash.ts

```diff
@@ -27,17 +27,29 @@ function hashLedgerHeader(ledgerHeader) {
 
 function computeTransactionHash(ledger, version,
     options: ComputeLedgerHashOptions) {
+  let transactions: any[]
   if (ledger.rawTransactions === undefined) {
     if (options.requireRawTransactions !== true) {
       return ledger.transactionHash
```

Comment:
Should `headerOnly` default to `true`?

Reply:
It looks like previously the presence of `rawState` and `rawTransactions` determined whether we would calculate the corresponding hashes (as opposed to just using the provided hashes to compute the hash of the ledger header).
Despite my earlier suggestion, I'm leaning towards continuing to do the same instead of adding `headerOnly`. The one change would be that we would look for either `rawTransactions` or `transactions`. I think the `ledger is missing raw transactions` ValidationError you added makes sense, since there isn't a way for `getLedger` to return something without either `rawTransactions` or `rawTransaction`s.